apps/dialtone-documentation/docs/components/icon.md

```diff
@@ -8,24 +8,166 @@ storybook: https://vue.dialpad.design/?path=/docs/components-icon--default
+
+```js
+<dt-stack class="d-fc-success">
+  <dt-icon name="settings" size="300" />
+  <p>Settings</p>
+</dt-stack>
+```
+
+</div>
+</div>
+
+<div class="d-d-grid d-gg24 d-g-cols3 md:d-g-cols1">
+
+When setting the color of an icon take these into considaration:
 
-<icon-catalog></icon-catalog>
+<div class="d-gc1">
+<div style="background: var(--dt-color-purple-100)" class="d-p16 d-hmn164 d-bar8 d-d-flex d-ai-center">
+<dt-stack direction="row" as="section" gap="100" class="d-bgc-primary d-bc-default d-bar32 d-py8 d-px16 d-w100p">
+<dt-stack direction="row" as="section" gap="300" class="d-fl1">
+<dt-icon name="headphones" size="300" ariaLabel="Headphones icon" />
+<p class="d-body-base d-truncate d-w100p d-wmx102">Ai Contact Center</p>
+</dt-stack>
+<dt-stack direction="row" as="section" gap="300">
+<dt-icon class="d-fc-success" name="bell" size="200" ariaLabel="Bell Icon" />
+<p class="d-fc-success d-body-small">Available</p>
+</dt-stack>
+</dt-stack>
+</div>
 
-## Variants and examples
+- Match the icon color with the text color when pairing them.
+- All icons are monochrome.
+
+</div>
+
+<div class="d-gc1">
+<div class="d-bgc-critical-subtle-opaque d-p16 d-hmn164 d-bar8 d-d-flex d-ai-center">
+<dt-stack direction="row" as="section" gap="100" class="d-bgc-primary d-bc-default d-bar32 d-py8 d-px16 d-w100p">
+<dt-stack direction="row" as="section" gap="300" class="d-fl1">
+<dt-icon name="headphones" size="300" ariaLabel="Headphones icon" />
+<p class="d-body-base d-truncate d-w100p d-wmx102">Ai Contact Center</p>
+</dt-stack>
+<dt-stack direction="row" as="section" gap="300">
+<dt-icon class="d-fc-critical" name="bell" size="200" ariaLabel="Bell Icon" />
+<p class="d-fc-success d-body-small">Available</p>
+</dt-stack>
+</dt-stack>
+</div>
+
+- Don’t use different colors for text and icons.
+- Don’t use more than one color within an icon.
+
+</div>
+</div>
+
+### Icon and Text aligment
+
+<div class="d-d-grid d-gg24 d-g-cols3 md:d-g-cols1">
+
+We encourage utilizing the [Stack component](/components/stack/) for aligning elements both horizontally and vertically.
+
+<div class="d-gc2">
+<code-well-header>
+  <div class="d-d-grid d-gg24 d-g-cols2 md:d-g-cols1 d-w100p">
+    <div class="d-fl-center">
+      <dt-stack :direction="selectedDirection" class="d-fl-center" gap="300">
+      <dt-icon name="settings" size="300" />
+      <p>Settings</p>
+      </dt-stack>
+    </div>
+    <dt-select-menu label="Direction" :options="stackDirection" @change="changeDirection" />
+  </div>
+</code-well-header>
+
+```js
+<dt-stack direction="row" class="d-fl-center" gap="300">
+  <dt-icon name="settings" size="300" />
+  <p>Settings</p>
+</dt-stack>
+```
+
+</div>
+</div>
+
+### Accessibility
+
+- If the icon serves a purpose beyond its visual representation, provide a clear description in the `ariaLabel` prop. This ensures all users understand its function, regardless of how they interact with it, e.g: `<dt-icon name="settings" ariaLabel="Edit your profile" />`
```

Comment:
kebab case is the standard styling for html attributes / vue props

`aria-label="Edit your profile"`

Reply:
Oh thanks! I got that from here: https://dialtone.dialpad.com/vue/index.html?path=/docs/components-icon--docs should we update it?

Reply:
With vue props when they are referred to in the template (as above) kebab case is used, however if they are used in javascript camel case is used. Storybook shows the javascript prop names, so what we see there is expected.

Reply:
I understand. Thank you!